ci/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM fedora
+MAINTAINER Igor Gnatenko <i.gnatenko.brain@gmail.com>
+
+WORKDIR /opt/rpm
+COPY . .
+
+RUN echo -e "deltarpm=0\ninstall_weak_deps=0\ntsflags=nodocs" >> /etc/dnf/dnf.conf
+RUN dnf -y update
+RUN dnf -y install \
```

Comment:
In some [PAPR](https://github.com/projectatomic/papr) projects like rpm-ostree we just pull the builddeps from the distro, I think this is a lot saner than repeating them here again.  See [this example](https://github.com/projectatomic/rpm-ostree/blob/f25444554dac95c5ff469f64192bad17a2916c3a/ci/build.sh#L20).